model/metrics/clr-metrics.yaml

```
@@ -0,0 +1,166 @@
+groups:
+  - id: metric.clr.gc.collections.count
+    type: metric
+    metric_name: clr.gc.collections.count
+    brief: "Number of garbage collections that have occurred since the process started."
```

```suggestion
    brief: "Number of garbage collections that have occurred since the process has started."
```

I feel that emphasising the relevance of the action to the present moment is important for metric
Also relevant for other `brief` with `process started`